augly/image/utils/bboxes.py

```diff
@@ -444,6 +444,31 @@ def get_enclosing_bbox(
     )
 
 
+def spatial_bbox_helper(
+    bbox: Tuple, src_w: int, src_h: int, aug_function: Callable, **kwargs
+) -> Tuple:
+    """
+    Computes the bbox that encloses a white box in a black background
+    for any augmentation.
+    """
+    dummy_image = create_test_image(w=src_w, h=src_h, bbox=bbox)
```

Comment:
`create_test_image()` is not defined here. Please create `dummy_image` here using the code from `create_test_image()` in the notebook.